tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-distributed-test-ci-pipeline.yml

```diff
@@ -0,0 +1,59 @@
+trigger: none
+
+jobs:
+- job: Onnxruntime_Linux_GPU_Distributed_Test
+
+  timeoutInMinutes: 120
+  pool: 'Linux-Multi-GPU-V100'
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  - task: Docker@2
+    displayName: login
+    inputs:
+      containerRegistry: onnxruntimebuildcache
+      command: login
+      addPipelineData: false
+
+  - script: |
+      tools/ci_build/github/linux/run_dockerbuild.sh \
```

Comment:
consider using tools/ci_build/github/azure-pipelines/templates/run-docker-build-steps.yml
for example: https://github.com/microsoft/onnxruntime/blob/ba739a80005dfd7eb1dbaac4d589d29c92393cfc/tools/ci_build/github/azure-pipelines/templates/linux-ci.yml#L37

Reply:
this also removes the need for specifying docker login/logout steps. those are included in the template

Reply:
Oh I see. I didn't realize that. Updating.